dbgen test uses deprecated -h HOST and -p PORT options for ldapmodify

**Issue Description**
OpenLDAP 2.6+ deprecated -h HOST and -p PORT options in all their command line tools. 
They are not allowed anymore and cause 'unknown option -' errors.
See https://bugs.openldap.org/show_bug.cgi?id=8618

dbgen test uses `ldapmodify` with deprecated -h HOST and -p PORT options. On F36 with OpenLDAP 2.6.x this test fails:
```
FAILED dirsrvtests/tests/suites/clu/dbgen_test.py::test_dsconf_dbgen_groups
FAILED dirsrvtests/tests/suites/clu/dbgen_test.py::test_dsconf_dbgen_cos_classic
FAILED dirsrvtests/tests/suites/clu/dbgen_test.py::test_dsconf_dbgen_cos_pointer
FAILED dirsrvtests/tests/suites/clu/dbgen_test.py::test_dsconf_dbgen_cos_indirect
FAILED dirsrvtests/tests/suites/clu/dbgen_test.py::test_dsconf_dbgen_cos_template
FAILED dirsrvtests/tests/suites/clu/dbgen_test.py::test_dsconf_dbgen_managed_role
FAILED dirsrvtests/tests/suites/clu/dbgen_test.py::test_dsconf_dbgen_filtered_role
FAILED dirsrvtests/tests/suites/clu/dbgen_test.py::test_dsconf_dbgen_nested_role
FAILED dirsrvtests/tests/suites/clu/dbgen_test.py::test_dsconf_dbgen_mod_ldif_mixed
FAILED dirsrvtests/tests/suites/clu/dbgen_test.py::test_dsconf_dbgen_nested_ldif
FAILED dirsrvtests/tests/suites/clu/dsctl_tls_test.py::test_tls_command_returns_error_text
```

Fix this by using -H URL option for client tools.

--- a/dirsrvtests/tests/suites/clu/dbgen_test.py
+++ b/dirsrvtests/tests/suites/clu/dbgen_test.py
@@ -56,7 +56,8 @@ def run_ldapmodify_from_file(instance, ldif_file, output_to_check=None):
     LDAP_MOD = '/usr/bin/ldapmodify'
     log.info('Add entries from ldif file with ldapmodify')
     result = subprocess.check_output([LDAP_MOD, '-cx', '-D', DN_DM, '-w', PASSWORD,
-                                      '-h', instance.host, '-p', str(instance.port), '-af', ldif_file])
+        '-H', f'ldap://{instance.host}:{instance.port}', '-af', ldif_file])
+
     if output_to_check is not None:
         assert output_to_check in ensure_str(result)
 

--- a/dirsrvtests/tests/suites/clu/dbgen_test_usan.py
+++ b/dirsrvtests/tests/suites/clu/dbgen_test_usan.py
@@ -74,7 +74,8 @@ def run_ldapmodify_from_file(instance, ldif_file, output_to_check=None):
     LDAP_MOD = '/usr/bin/ldapmodify'
     log.info('Add entries from ldif file with ldapmodify')
     result = subprocess.check_output([LDAP_MOD, '-cx', '-D', DN_DM, '-w', PASSWORD,
-                                      '-h', instance.host, '-p', str(instance.port), '-af', ldif_file])
+        '-H', f'ldap://{instance.host}:{instance.port}', '-af', ldif_file])
+
     if output_to_check is not None:
         assert output_to_check in ensure_str(result)
 

